### PR TITLE
Fix element selection

### PR DESCRIPTION
**Pull request by kaalleen**
As pointed out by @bigredfrog in #1304 the correction of the element selection was again not correct.
Next try here ;)

### CODE REVIEW

File: lib/extensions/base.py [hunk 1]
```diff
@@ -137,8 +137,7 @@ def descendants(self, node, selected=False, troubleshoot=False):  # noqa: C901
             if len(list(layer_commands(node, "ignore_layer"))):
                 return []
 
-        if (node.tag in [EMBROIDERABLE_TAGS, SVG_GROUP_TAG] and
-                element.get_style('display', 'inline') is None):
+        if (node.tag in EMBROIDERABLE_TAGS or node.tag == SVG_GROUP_TAG) and element.get_style('display', 'inline') is None:
```
Comment (bigredfrog) [inline lib/extensions/base.py:140]: Are you enforcing line length limits at all in code base?

Noting I ran this specific change locally and my stitch plan looks sane. 

Only the same 7 color blocks and elements I expect

**Comment by bigredfrog** [comment]
Also tested for pdf export, and sane end to end.

**Review by bigredfrog — APPROVED** [review]
I am happy to approve this and leave my question on formating of line length dangaling, assumign we would have pep8 enforcement or similar if desired.

The fix looks real good from my independant testing.

**Comment by kaalleen** [comment]
We run `flake8 . --count --max-complexity=10 --max-line-length=150 --statistics --exclude=pyembroidery,__init__.py,electron,build,src` to check the code. You can find it in the Makefile.

Max line length is ok in our case.

**Pull request merged**